code/game/machinery/Sleeper.dm

```diff
@@ -23,6 +23,7 @@
 	var/initial_bin_rating = 1
 	var/min_health = -25
 	var/controls_inside = FALSE
+	var/autoeject_dead = FALSE
```

Comment:
Minor styling. it reads a bit odd with auto and eject being put right after another. Putting a _ in between will improve readability